frontend/apps/web/package.json

```diff
@@ -64,6 +65,7 @@
     "react-hook-form": "^7.52.2",
     "react-icons": "^5.2.1",
     "recharts": "^2.12.7",
+    "snapsvg-cjs": "^0.0.6",
```

Comment:
Do we need this or is it being used anywhere? If not, let's remove.

Reply:
not seeing it used anywhere - removing it